RELEASE_NOTES.md

```diff
@@ -0,0 +1,99 @@
+# Common Theme Release workflow
+
+## Release Notes
+- Frequency of releases
+    - Major release: rare (Example: paradigm change)
+    - Minor release: monthly (Example: breaking changes, documentation, fixes)
+    - Patch release: as and when (Example: small features)
```

Comment:
This is not semver - I'm fine with that, and don't think it's a problem, but wanted to call this out specifically to make sure we're all on the same page there